src/shared/copy.c

```diff
@@ -1096,12 +1096,12 @@ static int fd_copy_directory(
                 if (dot_or_dot_dot(de->d_name))
                         continue;
 
-                r = look_for_signals(copy_flags);
```

Comment:
Hm, we usually use `r` as storing error code temporally, and `ret` for gathered error code. I'd like to follow the (undocumented) rule here.

Reply:
yes, please